quantum/plugins/ibm/accelerator/IBMAccelerator.cpp

```diff
@@ -176,6 +176,88 @@ bool hasMidCircuitMeasurement(
   return false;
 }
 
+bool IBMAccelerator::verifyJobsLimit(std::string& curr_backend) {
+  
+  // Get backend jobs limit
+  std::string getJobsLimitPath = "/api/Network/" + hub + "/Groups/" + group +
+                        "/Projects/" + project + "/devices/" + curr_backend +
+                        "/jobsLimit";
+  auto backend_jobslimit_response = get(IBM_API_URL, getJobsLimitPath, {},
+        {std::make_pair("version", "1"),
+        std::make_pair("access_token", currentApiToken)});
+  auto backend_jobslimit_response_json = json::parse(backend_jobslimit_response);
+
+  auto maximumJobs = backend_jobslimit_response_json["maximumJobs"].get<int>();
```

Comment:
I failed to run the example b/c `verifyJobsLimit` always failed if `maximumJobs` is -1.
It could be due to my specific account provider (not `open`) which doesn't have any limitation.
I suggest that the condition is changed to `return maximumJobs < 0 || runningJobs < maximumJobs;` to be more robust.